gremlin/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: gremlin
-version: 0.4.8
+version: 0.5.0
```

Comment:
Let's release this first as a "development version" (or "pre-release" version). The `-alpha` suffix tells helm that this version should not be installed unless someone supplies the `--devel` flag when installing (see [helm install](https://helm.sh/docs/helm/helm_install/)). This will allow us to install from the published repository in staging before we release everything to production.

Reply:
```suggestion
version: 0.5.0-alpha
```